### PR TITLE
Update README for KSQL 4.1

### PR DESCRIPTION
**Pull request by miguno**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,31 @@
 # ![KSQL rocket](ksq-lrocket.png) KSQL - Streaming SQL for Apache Kafka
 
-> **Important:** We recently [announced the General Availability](https://www.confluent.io/press-release/confluent-makes-ksql-available-confluent-platform-announces-general-availability/) of KSQL. It will be available for download in early April at http://confluent.io/ksql. Until then you can download the latest KSQL Developer Preview release at https://github.com/confluentinc/ksql/releases.
+> **KSQL is now GA and officially supported by Confluent. [Get started with KSQL today](#getting-started).**
```
Comment (joel-hamill) [inline README.md]: `Confluent Platform` or `Confluent Inc`?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
 
-- [KSQL Quick Start](https://github.com/confluentinc/ksql/tree/v0.5/docs/quickstart#quick-start): Demonstrates a simple workflow using KSQL to write streaming queries against data in Kafka.
-- [Clickstream Analysis Demo](https://github.com/confluentinc/ksql/tree/v0.5/ksql-clickstream-demo#clickstream-analysis): Shows how to build an application that performs real-time user analytics.
+
+<a name="getting-started"></a>
+# Getting Started and Download
+
+1. [Download KSQL](https://www.confluent.io/download/), which is included in the Enterprise and Open Source editions
+   of Confluent Platform.
+2. Follow the [Quickstart](https://docs.confluent.io/current/quickstart.html).
```
Comment (joel-hamill) [inline README.md]: `Quick Start` should be written as two-words (see [Style Guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-Terminology))

File: README.md [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
+
+We also provide further [KSQL Tutorials and Examples](https://docs.confluent.io/current/ksql/docs/tutorials/), including Docker-based variants.
```
Comment (joel-hamill) [inline README.md]: avoid using "we" or "our", instead use the second person "you" (per [Style Guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-VoiceandTone)). suggested:

```
For more information, see the [KSQL Tutorials and Examples](https://docs.confluent.io/current/ksql/docs/tutorials/), which includes 
Docker-based variants.
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,31 @@
...
+
+
+# Documentation
+
+Head over to the [KSQL documentation](https://docs.confluent.io/current/ksql/docs/).
```
Comment (joel-hamill) [inline README.md]: suggest: `You can find the KSQL documentation at [https://docs.confluent.io/current/ksql/docs/](https://docs.confluent.io/current/ksql/docs/).`

File: README.md [hunk 5]
```diff
@@ -83,22 +98,25 @@ CREATE TABLE error_counts AS
   -- focus on community-raised issues and requests (369 pull requests, 50 closed issues)
 
 
-# Documentation
-You can [find the KSQL documentation here](https://github.com/confluentinc/ksql/tree/v0.5/docs#ksql-documentation)
-
 # Join the Community
-Whether you need help, want to contribute, or are just looking for the latest news, you can find out how to [connect with your fellow Confluent community members here](https://www.confluent.io/contact-us-thank-you/).
+
+Whether you need help, want to contribute, or are just looking for the latest news, we encourage you to [connect with your fellow Confluent community members](https://www.confluent.io/contact-us-thank-you/).
```
Comment (joel-hamill) [inline README.md]: suggest:

```
You can get help, contribute to KSQL, or find the latest news by 
[connecting with the Confluent community](https://www.confluent.io/contact-us-thank-you/).
```

File: README.md [hunk 6]
```diff
@@ -83,22 +98,25 @@ CREATE TABLE error_counts AS
...
 * Join the [Confluent Google group](https://groups.google.com/forum/#!forum/confluent-platform).
 
+
 # Contributing
+
 Contributions to the code, examples, documentation, etc, are very much appreciated. For more information, see the [contribution guidelines](/docs/contributing.md).
 
 - Report issues and bugs directly in [this GitHub project](https://github.com/confluentinc/ksql/issues).
-- Read the [Developer notes](./DEVELOPMENT.md)
+- Read the [Developer Notes](./DEVELOPMENT.md) on how to work with the KSQL source code, including building and
```
Comment (joel-hamill) [inline README.md]: suggest:

```
For more information about how to work with the KSQL source code, 
including building and testing KSQL, see the [Developer Notes](./DEVELOPMENT.md).
```

File: README.md (hunk 1 above)
Comment (miguno) [reply README.md]: Good point.  I picked `Confluent Inc.`.

File: README.md (hunk 2 above)
Comment (miguno) [reply README.md]: Ack

File: README.md (hunk 3 above)
Comment (miguno) [reply README.md]: Ack

File: README.md (hunk 5 above)
Comment (miguno) [reply README.md]: Ack

**Comment by miguno** [comment]
Updated, please review again.

**Pull request merged**